plugins/00_dokku-standard/commands

```diff
@@ -10,27 +10,27 @@ case "$1" in
     pushd "$TMP_WORK_DIR" &> /dev/null
 
     case "$IMAGE_SOURCE_TYPE" in
-      buildstep)
+      herokuish)
         id=$(tar -c . | docker run -i -a stdin $DOKKU_IMAGE /bin/bash -c "mkdir -p /app && tar -xC /app")
         test "$(docker wait $id)" -eq 0
         docker commit $id $IMAGE > /dev/null
         [[ -d $CACHE_DIR ]] || mkdir $CACHE_DIR
         # *DEPRECATED* in v0.4.0: `pluginhook pre-build` will be removed in future releases
         pluginhook pre-build "$APP"
-        pluginhook pre-build-buildstep "$APP"
+        pluginhook pre-build-herokuish "$APP"
```

Comment:
I can go through the pr if you'd like and point out all the other hooks. TBH I was being lazy and should have done it myself, but given the rebase i figured you'd just handle it all in one :)